transform/uuid.go

```diff
@@ -30,7 +30,10 @@ func UUID(spec *Config, data []byte) ([]byte, error) {
 
 		switch version {
 		case 4:
-			u = uuid.NewV4()
+			u, err = uuid.NewV4()
+			if err != nil {
+				return nil, err
```

Comment:
Yeah, I probably should have included my thoughts about that in the PR. The previous behavior of the uuid library was to panic if there was an error getting random bytes, so actually returning an error does seem like an improvement. Other comments on the issue you linked (satori/go.uuid#66) hinted at retrying, but I'm skeptical of that. The only time I've actually encountered an entropy pool issue it seemed like there was very little entropy being generated and the pool wasn't filling up in a reasonable amount of time, which would mean (in that particular case at least) retrying wouldn't help much. With the Go stdlib specifically, I'm not actually sure if it would just block indefinitely or error out (https://golang.org/src/crypto/rand/rand_linux.go, https://golang.org/src/crypto/rand/rand.go). As a library user, I think I would prefer a very rare traceable error over a very rare inscrutable hang.

Reply:
Good point. That seems reasonable.